### PR TITLE
Adding Links

### PR DESCRIPTION
**Pull request by Math3v**

### CODE REVIEW

File: lib/reddex/links/link.ex [hunk 1]
```diff
@@ -0,0 +1,22 @@
+defmodule Reddex.Links.Link do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+
```
Comment (Math3v) [inline lib/reddex/links/link.ex]: There should be no more than 1 consecutive blank lines.

File: lib/reddex/links/link.ex [hunk 2]
```diff
@@ -0,0 +1,22 @@
+defmodule Reddex.Links.Link do
```
Comment (Math3v) [inline lib/reddex/links/link.ex]: Modules should have a @moduledoc tag.

**Comment by Math3v** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 2 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/PrimeHammer/reddex/pulls/11.

File: lib/reddex/links/link.ex [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    |> cast(attrs, [:url, :tags_input, :title, :description])
+    |> validate_required([:url, :tags_input])
+    |> unique_constraint(:url)
+    |> tags_input_to_tags_array()
+  end
+
+  defp tags_input_to_tags_array(changeset) do
+    case changeset do
+      %Ecto.Changeset{valid?: true, changes: %{tags_input: tags_input}} ->
+        tags =
```
Comment (Math3v) [inline lib/reddex/links/link.ex]: There should be no trailing white-space at the end of a line.

**Comment by Math3v** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/PrimeHammer/reddex/pulls/11.

**Comment by Math3v** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/PrimeHammer/reddex/pulls/11.

File: lib/reddex/links/link.ex (hunk 2 above)
Comment (erich) [reply lib/reddex/links/link.ex]: @Math3v i'd say it's fine. the comment stays here even if the credo doesn't show it on localhost.

**Pull request merged**